pyproject.toml

```diff
@@ -87,6 +87,7 @@ lint = [
   "pre-commit-uv==4.1.4",
 ]
 
+[dependency-groups]
 dev = [
   "any-agent[docs,tests,lint]",
   "pre-commit",
```

Comment:
Duplicated from `lint`

```suggestion
```

Reply:
I've updated the `pyproject.toml` now to configure the groups (and removed `pre-commit` from the 'dev' one directly. 😄